package.json

```diff
@@ -24,7 +24,9 @@
     "index.js",
     "typings/index.d.ts"
   ],
-  "dependencies": {},
+  "dependencies": {
+    "omelette": "^0.4.12"
```

Comment:
added production dependency to omelette, which target supports for node 5+, I guess it should be ok?